README.md

```diff
@@ -59,3 +59,52 @@ workflows:
             tags:
               only: /v.*/
 ```
+
+### Public Cache
+
+This orb can sync the buildroot dependencies to a public cache in AWS S3.
+Perform the following steps to enable the public cache for a Nerves system.
+
+- Add `save-public-cache: true` to the `build-tools/get-br-dependencies` job.
+
+```yaml
+workflows:
+  version: 2
+  build_test_deploy:
+    jobs:
+      - build-tools/get-br-dependencies:
+          exec:
+            <<: *exec
+          save-public-cache: true
+```
+
+- Create an AWS S3 bucket.
+- Set the bucket policy to allow read-only access for anonymous users.
+
+```json
+{
+  "Version":"2012-10-17",
+  "Statement":[
+    {
+      "Sid":"PublicRead",
+      "Effect":"Allow",
+      "Principal": "*",
+      "Action":["s3:GetObject","s3:GetObjectVersion"],
+      "Resource":["arn:aws:s3:::<bucket-name>/*"]
+    }
+  ]
+}
+```
+
+- In the AWS dashboard, create an AWS access key for the AWS user account that
+  CI will use to access the S3 bucket.
+
+- Add the following environment variables to the CircleCI project for the
+  Nerves system:
+  - `AWS_ACCESS_KEY_ID`
+  - `AWS_SECRET_ACCESS_KEY`
+  - `AWS_REGION`
+  - `CACHE_BUCKET_URI` (`s3://<bucket-name>/`)
+
+- In your Nerves system `nerves_defconfig` file, set the buildroot cache to the
+  S3 bucket URL: `BR2_PRIMARY_SITE="https://<bucket-name>.s3.amazonaws.com"`
```

Comment:
I'd like CI to set `BR2_PRIMARY_SITE` like before. The reason is that I'd prefer users to hit the Nerves package download site as the backup like what the Buildroot project does. Of course, the Nerves project download site needs to be populated before that's possible.

Reply:
`BR2_PRIMARY_SITE` set in be69d4a.